src/vm/machine.lisp

```diff
@@ -1 +1,91 @@
 (in-package :cl-braces.vm.machine)
+
+(defparameter *debug-execution* nil)
```

Comment:
Let’s replace this with a feature instead. Maybe cl-debug? Or maybe flip it an have no-debug)